pkg/client/clientcmd/api/types.go

```diff
@@ -83,6 +83,8 @@ type Context struct {
 	AuthInfo string `json:"user"`
 	// Namespace is the default namespace to use on unspecified requests
 	Namespace string `json:"namespace,omitempty"`
+	// NamespacePath is the path to a kubernetes ns file (~/.kubernetes_ns)
```

Comment:
We need to add a comment somewhere in here that we want to nuke this.  I'd prefer it be on client_config.go.  We also need a decision on when we can nuke backwards compatibility with .kubernetes_ns now that we have context, and that all the use cases are covered.  @brendandburns do you think we could reasonably just nuke namespace-path right now?